docs/source/openapi.json

```diff
@@ -6105,7 +6132,132 @@
                             }
                           }
                         }
-                      ]
+                      ],
+                      "partial": false
+                    }
+                  },
+                  "A split (lmsys/lmsys-chat-1m) with a boolean column": {
+                    "summary": "Statistics on a boolean column 'redacted'.",
+                    "description": "Try with https://datasets-server.huggingface.co/statistics?dataset=lmsys/lmsys-chat-1m&config=default&split=train.",
```

Comment:
Looks like `lmsys/lmsys-chat-1m` is a gated dataset, maybe point to a public one so that it is easier for the users to see the data?